Add docs `Label` widget. (#146)
yiisoft/form: docs/label.md

use Yiisoft\Form\Widget\Field;
use Yiisoft\Form\Widget\Form;
use Yiisoft\Form\Widget\Label;
use Yiisoft\Form\Widget\Text;

/**
 * @var FormModelInterface $data
 * @var object $csrf
 */
?>

<?= Form::widget()->action('widgets')->csrf($csrf)->begin() ?>
    <?= Label::widget()->config($formModel, 'name') ?>
    <?= Text::widget()->config($formModel, 'name') ?>
    <hr class="mt-3">
    <?= Field::widget()->submitButton(['class' => 'button is-block is-info is-fullwidth', 'value' => 'Save']) ?>
<?= Form::end() ?>
```

That would generate the following code:

```html
<form action="widget" method="POST" _csrf="A6DW7BkU5HOsjrVRbr9jFO32ZulADJaTYS0q3G1n40gwl6-CfECnQ9vY7SgP5wJMm8FXkwNW-ecnSnjoJCapIw==">
    <input type="hidden" name="_csrf" value="A6DW7BkU5HOsjrVRbr9jFO32ZulADJaTYS0q3G1n40gwl6-CfECnQ9vY7SgP5wJMm8FXkwNW-ecnSnjoJCapIw==">
    <label for="testform-name">First name:</label>
    <input type="text" id="testform-name" name="TestForm[name]">
    <hr class="mt-3">
    <div>
        <input type="submit" id="submit-186948574010001" class="button is-block is-info is-fullwidth" name="submit-186948574010001" value="Save">
    </div>
</form>
```

### Custom label text

> To configure a custom label text, we simply change it in the widget, taking the previous example the code would be the following: 

Reviewer: ```suggestion
To configure a custom label text, you can specify it when the widget is used: 
```

    <?= Label::widget()->config($formModel, 'name')->label('Name:') ?>
    <?= Text::widget()->config($formModel, 'name') ?>
    <hr class="mt-3">
    <?= Field::widget()->submitButton(['class' => 'button is-block is-info is-fullwidth', 'value' => 'Save']) ?>
<?= Form::end() ?>
```

That would generate the following code:

```html
<form action="widgets" method="POST" _csrf="dlD6bhQ1cOW8W5UC5SkHKWBPjxijMkEkx9-10_nKta9FZ4MAcWEz1csNzXuEcWZxFni-YuBoLlCBuOfnsIv_xA==">
    <input type="hidden" name="_csrf" value="dlD6bhQ1cOW8W5UC5SkHKWBPjxijMkEkx9-10_nKta9FZ4MAcWEz1csNzXuEcWZxFni-YuBoLlCBuOfnsIv_xA==">
    <label for="testform-name">Name:</label>
    <input type="text" id="testform-name" name="TestForm[name]">
    <hr class="mt-3">
    <div>
        <input type="submit" id="submit-188768911791001" class="button is-block is-info is-fullwidth" name="submit-188768911791001" value="Save">
    </div>
</form>
```

### `Label` methods:

Method | Description | Default
-------|-------------|---------
`config(FormModelInterface $formModel, string $attribute, array $attributes = [])` | Configures the widget. |
`encode(bool $value)` | Whether to encode the error message. | `true`

> `for(?string $value)` | The id of a labelable form-related element in the same document as the tag label element. | `''`

Reviewer: ```suggestion
`for(?string $value)` | The ID of a form element to associate label with. | `''`
```
